Tests/iaas/image-metadata/image-md-check.py

```diff
@@ -217,11 +329,45 @@ def report_stdimage_coverage(imgs):
     return err
 
 
+def miss_replacement_images(images, outd_list):
+    """Go over list of images to find replacement imgs for outd_list, return the ones that are left missing"""
+    rem_list = []
+    for outd in outd_list:
+        success = False
+        last_spc = outd.rfind(" ")
+        shortnm = outd
+        if last_spc != -1:
+            shortnm = outd[:last_spc]
+        for imgnm in images:
+            # Skip over other images
+            if imgnm != outd and imgnm != shortnm:
+                continue
+            # Skip over itself
+            if imgnm == outd:  # or success:
+                continue
+            img = conn.image.find_image(imgnm)
```

Comment:
I know this script can be invoked with a list of image names, but usually it will check every image, and so it will fetch the image info for every image. Wouldn't it be better to do this once, with a single API call (maybe multiple, but few calls if paging is involved)?